### PR TITLE
Add postfix completion for let else

### PR DESCRIPTION
**Pull request by Alainx277**
Adds a postfix completion for let else syntax, similar to the if let postfix.

### CODE REVIEW

File: crates/ide-completion/src/completions/postfix.rs [hunk 1]
```diff
@@ -84,6 +89,17 @@ pub(crate) fn complete_postfix(
                 )
                 .add_to(acc, ctx.db);
 
+                postfix_snippet(
+                    "lete",
+                    "let Ok else {}",
+                    &if in_loop {
+                        format!("let Ok($1) = {receiver_text} else {{\n    ${{2|continue,break,return|}};\n}};\n$0")
+                    } else {
+                        format!("let Ok($1) = {receiver_text} else {{\n    return;\n}};\n$0")
```
Comment (Veykril) [inline crates/ide-completion/src/completions/postfix.rs:98]: We should make the `return` here also a snippet I think

Comment (Alainx277) [reply crates/ide-completion/src/completions/postfix.rs]: Good idea

**Review by saiintbrisson — COMMENTED** [review]
Super nice PR and auto completions. What do you think of adding the postfix for non-try types as well?

**Comment by Alainx277** [comment]
> Super nice PR and auto completions. What do you think of adding the postfix for non-try types as well?

Thanks. I think it could be useful, but its also doesn't exist for `if let` yet. Maybe you could make a follow-up PR that adds it for both? It would be nice if the enum variants were placeholders (like "continue, break, return" in this PR) 👍

**Comment by Alainx277** [comment]
@Veykril sorry for the ping, but I was wondering if this MR will get another review? I've addressed the last review over 2 months ago and would love for this small improvement to make it into RA.

**Comment by Veykril** [comment]
Apologies I just don't have time on my hands to go through the PR backlog until january.

File: crates/ide-completion/src/completions/postfix.rs [hunk 2]
```diff
@@ -84,6 +89,17 @@ pub(crate) fn complete_postfix(
...
                 .add_to(acc, ctx.db);
 
+                postfix_snippet(
+                    "lete",
+                    "let Ok else {}",
+                    &if in_loop {
+                        format!("let Ok($1) = {receiver_text} else {{\n    ${{2|continue,break,return|}};\n}};\n$0")
+                    } else {
+                        format!("let Ok($1) = {receiver_text} else {{\n    ${{2:return}};\n}};\n$0")
+                    },
```
Comment (Veykril) [inline crates/ide-completion/src/completions/postfix.rs:99]:
```suggestion
                    &format!("let Ok($1) = {receiver_text} else {{\n    $0\n}};"),
```
Let's not prefill these, there are more options people could strife for. Same for the option one below

Comment (Alainx277) [reply crates/ide-completion/src/completions/postfix.rs]: I'm a bit sad to lose the convenience, but I see how these could be annoying when you want to do more complex things.

**Comment by Veykril** [comment]
Thanks!
@bors r+

**Pull request merged**